index.js

```diff
@@ -1,21 +1,38 @@
 'use strict';
 
-module.exports = {
-  env: {
-    node: true,
-  },
-  extends: [
-    'crowdstrike',
-    'plugin:n/recommended',
-  ],
-  plugins: [
-    'json-files',
-  ],
-  rules: {
-    'strict': 'error',
+const jsonFiles = require('eslint-plugin-json-files');
+const globals = require('globals');
+const { FlatCompat } = require('@eslint/eslintrc');
```

Comment:
eventually can get rid of `FlatCompat` once we update https://github.com/CrowdStrike/eslint-config-crowdstrike to flat configs as well